dashboard/app/views/plc/enrollment_task_assignments/complete/_learning_resource_task.html.haml

```diff
@@ -0,0 +1,4 @@
+%p Hello, for this task, just check out this website
+
+%p
+  %iframe{width: '800px', height: '800px', src: 'http://' + @task.resource_url}
```

Comment:
Using string interpolation here will prevent an exception if `@task.resource_url` is nil.  Another issue you may run into is that you can't load `http://` iframes inside an `https://` page due to content security restrictions in the browser.

(One more instance in `_learning_resource_task.html.haml`)